May news

aeslaughter opened the pull request:
@idaholab/moose-ccb Please add your contribution for April, please include small changes and minor enhancements as well. You can also add things already completed to the May document to help along the next update.

refs #11447

aeslaughter committed:
May news

(refs #11447)

moosebuild commented:
Job [Documentation](https://civet.inl.gov/job/739662/) on 9ba11c8 wanted to post the following:

View the site [here](https://mooseframework.inl.gov/docs/PRs/17811/site)

This comment will be updated on new commits.

GiudGiud commented:
Nothing to review yet ^^.
I ll make a PR for pinsfv

GiudGiud committed:
Add my May contributions

roystgnr committed:
Add libmesh-level changes from the Apr 7 update

lindsayad committed:
Add finite volume interface kernel blurb

Also move PINSFV to the top because I think that's the most interesting
new capability

roystgnr committed:
Mention MeshGenerator::addSubGenerator

roystgnr committed:
Move April changes to May newsletter

aeslaughter committed:
Fix doc syntax mistake

(refs #11447)

dewenyushu committed:
Add new CP class in the News

aeslaughter committed:
Merge pull request #32 from dewenyushu/may_newsletter

Add new CP class in the News
